src/Aspire.Hosting.Docker/DockerComposeEnvironmentContext.cs

```diff
@@ -81,12 +81,15 @@ private void ProcessVolumes(DockerComposeServiceResource serviceResource)
                 throw new InvalidOperationException("Volume source and target must be set");
             }
 
-            var source = mount.Source;
-            var name = mount.Source;
+            // For Docker Compose, use RelativeSource if available (keeps paths relative to compose file),
+            // otherwise fallback to Source (resolved absolute path)
+            var sourceForCompose = mount.RelativeSource ?? mount.Source;
```

Comment:
Why is this "forCompose"?

It might be simpler if we just have:

```C#
var source = mount.RelativeSource ?? mount.Source;
var name = source;
```